assets/xml/archives/schedule_dma_static.xml

```diff
@@ -1,28 +1,28 @@
+        <Texture Name="gBombersNotebookPhotoBombShopLadyTex" OutName="bombers_notebook_photo_bomb_shop_lady" Format="rgba16" Width="32" Height="32" Offset="0x1800" />
+        <Texture Name="gBombersNotebookPhotoRomaniTex" OutName="bombers_notebook_photo_romani" Format="rgba16" Width="32" Height="32" Offset="0x2000" />
+        <Texture Name="gBombersNotebookPhotoCremiaTex" OutName="bombers_notebook_photo_cremia" Format="rgba16" Width="32" Height="32" Offset="0x2800" />
+        <Texture Name="gBombersNotebookPhotoMayorDotourTex" OutName="bombers_notebook_photo_mayor_dotour" Format="rgba16" Width="32" Height="32" Offset="0x3000" />
+        <Texture Name="gBombersNotebookPhotoMadameAromaTex" OutName="bombers_notebook_photo_madame_aroma" Format="rgba16" Width="32" Height="32" Offset="0x3800" />
+        <Texture Name="gBombersNotebookPhotoTotoTex" OutName="bombers_notebook_photo_toto" Format="rgba16" Width="32" Height="32" Offset="0x4000" />
+        <Texture Name="gBombersNotebookPhotoGormanTex" OutName="bombers_notebook_photo_gorman" Format="rgba16" Width="32" Height="32" Offset="0x4800" />
+        <Texture Name="gBombersNotebookPhotoPostmanTex" OutName="bombers_notebook_photo_postman" Format="rgba16" Width="32" Height="32" Offset="0x5000" />
+        <Texture Name="gBombersNotebookPhotoRosaSistersTex" OutName="bombers_notebook_photo_rosa_sisters" Format="rgba16" Width="32" Height="32" Offset="0x5800" />
+        <Texture Name="gBombersNotebookPhotoToiletHandTex" OutName="bombers_notebook_photo_toilet_hand" Format="rgba16" Width="32" Height="32" Offset="0x6000" />
+        <Texture Name="gBombersNotebookPhotoAnjusGrandmotherTex" OutName="bombers_notebook_photo_anjus_grandmother" Format="rgba16" Width="32" Height="32" Offset="0x6800" />
+        <Texture Name="gBombersNotebookPhotoKamaroTex" OutName="bombers_notebook_photo_kamaro" Format="rgba16" Width="32" Height="32" Offset="0x7000" />
+        <Texture Name="gBombersNotebookPhotoGrogTex" OutName="bombers_notebook_photo_grog" Format="rgba16" Width="32" Height="32" Offset="0x7800" />
+        <Texture Name="gBombersNotebookPhotoGormanBrothersTex" OutName="bombers_notebook_photo_gorman_brothers" Format="rgba16" Width="32" Height="32" Offset="0x8000" />
+        <Texture Name="gBombersNotebookPhotoShiroTex" OutName="bombers_notebook_photo_shiro" Format="rgba16" Width="32" Height="32" Offset="0x8800" />
+        <Texture Name="gBombersNotebookPhotoGuruGuruTex" OutName="bombers_notebook_photo_guru_guru" Format="rgba16" Width="32" Height="32" Offset="0x9000" />
+        <Texture Name="gBombersNotebookPhotoBombersTex" OutName="bombers_notebook_photo_bombers" Format="rgba16" Width="32" Height="32" Offset="0x9800" />
+        <Texture Name="gBombersNotebookPhotoMadameAromaUnusedTex" OutName="bombers_notebook_photo_madame_aroma_unused" Format="rgba16" Width="32" Height="32" Offset="0xA000" />
```

Comment:
Is `gBombersNotebookPhotoMadameAromaUnusedTex` identical to `gBombersNotebookPhotoMadameAromaTex`?

Reply:
Not quite. It is considerably brighter (making it look blurry), and it looks like a slightly different pose. (Regarding your second comment I'm actually not sure what to name it besides unused since it isn't just an identical copy).

Reply:
Actually went with just `Bright` to help distinguish it.